docs/reference/glossary.rst

```diff
@@ -127,6 +127,13 @@ Glossary
       RBNO is enabled by default for a subset node operations. 
       See :doc:`Repair Based Node Operations </operating-scylla/procedures/cluster-management/repair-based-node-operation>` for details.
 
+    RF-rack-valid keyspace
+      A keyspace with :doc:`tablets </architecture/tablets>` enabled is RF-rack-valid if all of its data centers
+      have the :term:`Replication Factor (RF) <Replication Factor (RF)>` of 0, 1, or the number of racks with at least
+      one :doc:`non-zero-token node </architecture/zero-token-nodes>` in that data center.
```

Comment:
why is the `non-zero-token node ` relevant ? its just another example for 0 RF in a DC.